Makefile

```diff
@@ -214,3 +214,38 @@ docs: mermaid-init.js mermaid.min.js ## 📚 Generate the documentation
 
 docs-serve: mermaid-init.js mermaid.min.js ## 📚 Generate and serve the documentation
 	mdbook serve --open
+
+
```

Comment:
Please move this to https://github.com/lambdaclass/ethrex/blob/main/tooling/sync/Makefile